google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplTest.java

```diff
@@ -1302,18 +1302,21 @@ public void testSignUrlWithOptions() throws NoSuchAlgorithmException, InvalidKey
   }
 
   @Test
-  public void testSignUrlForBlobWithSpecialChars() throws NoSuchAlgorithmException, InvalidKeyException,
-          SignatureException, UnsupportedEncodingException {
-    // List of chars under test were taken from https://en.wikipedia.org/wiki/Percent-encoding#Percent-encoding_reserved_characters
-    char[] specialChars = new char[]{'!','#','$','&','\'','(',')','*','+',',',':',';','=','?','@','[',']'};
+  public void testSignUrlForBlobWithSpecialChars() throws NoSuchAlgorithmException,
+          InvalidKeyException, SignatureException, UnsupportedEncodingException {
```

Comment:
I still see too much space here, is your IDE using tabs for indentation? Maybe try to switch to spaces and redo the copy pasting? (I appreciate)